test/test_suite.bats

```diff
@@ -92,56 +92,56 @@
 @test "Top Clients (descending, default)" {
   run bash -c 'echo ">top-clients >quit" | nc -v 127.0.0.1 4711'
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[1]} == "0 10 127.0.0.1 " ]]
+  [[ ${lines[1]} == "0 10 127.0.0.1 "* ]]
   [[ ${lines[2]} == "" ]]
 }
 
 @test "Top Clients (ascending)" {
   run bash -c 'echo ">top-clients asc >quit" | nc -v 127.0.0.1 4711'
   printf "%s\n" "${lines[@]}"
-  [[ ${lines[1]} == "0 10 127.0.0.1 " ]]
+  [[ ${lines[1]} == "0 10 127.0.0.1 "* ]]
   [[ ${lines[2]} == "" ]]
 }
 
 @test "Top Domains (descending, default)" {
   run bash -c 'echo ">top-domains >quit" | nc -v 127.0.0.1 4711'
   printf "%s\n" "${lines[@]}"
   [[ ${lines[1]} == "0 2 google.com" ]]
-  [[ ${lines[2]} == "1 1 version.ftl" ]]
-  [[ ${lines[3]} == "2 1 version.bind" ]]
-  [[ ${lines[4]} == "3 1 whitelisted.com" ]]
-  [[ ${lines[5]} == "4 1 regexa.com" ]]
-  [[ ${lines[6]} == "5 1 ftl.pi-hole.net" ]]
+  [[ "${lines[@]}" == *" 1 version.ftl"* ]]
+  [[ "${lines[@]}" == *" 1 version.bind"* ]]
+  [[ "${lines[@]}" == *" 1 whitelisted.com"* ]]
+  [[ "${lines[@]}" == *" 1 regexa.com"* ]]
+  [[ "${lines[@]}" == *" 1 ftl.pi-hole.net"* ]]
```

Comment:
Here and below: It is not meaningful to assume a particular order here as the values are sorted before output. It is unpredictable in which order they may come out. While this has always been the same when compiling for `glibc`, the new `musl` build reveals that another library may have a different interpretation here.